src/cloudWatchLogs/cloudWatchLogsUtils.ts

```diff
@@ -64,7 +62,7 @@ export function parseCloudWatchLogsUri(uri: vscode.Uri): {
  */
 export function canShowPreviousLogs(uri: vscode.Uri): boolean {
     const params = parseCloudWatchLogsUri(uri).parameters
-    return params.filterPattern ? false : true
+    return params.filterPattern === undefined ? true : false
```

Comment:
Can't we just do something like:
```suggestion
    return params.filterPattern === undefined
```

Reply:
Yep, I think I overthought that one haha.